Firebase/Auth/Source/Auth/FIRAuth.m

```diff
@@ -1136,10 +1136,17 @@ - (void)sendSignInLinkToEmail:(nonnull NSString *)email
           kMissingEmailInvalidParameterExceptionReason];
     }
 
+#if TARGET_OS_WATCH
+    /*
+     The sign in can't happen fully in-app, i.e. watchOS does not support universal links but watchOS has a mail client that will open this link appropriately
+     in a browser. From the browser you trigger a cloud function which sends a push notification to complete the sign-in.
```

Comment:
Is that something manual that you set up, or are you saying that's how it'll work automatically?

Reply:
Please see: https://github.com/firebase/firebase-ios-sdk/issues/4621#issuecomment-571914715

Reply:
@renkelvin can you please take a look at this bit specifically?

I'm comfortable merging the email/password sign in changes now, I'm not sure about the email link and will defer to @renkelvin for Auth's view of this (see the comment for the full solution and context).

Reply:
This looks good to me. Maybe add a link to https://github.com/firebase/firebase-ios-sdk/issues/4621#issuecomment-571914715 to make it easier to other developers?